autogen/outputs.tf

```diff
@@ -78,6 +78,13 @@ output "ca_certificate" {
   value       = "${local.cluster_ca_certificate}"
 }
 
+{% if private_cluster %}
+output "enable_binary_authorization" {
+description = "Enable BinAuthZ Admission controller"
+value = "${var.enable_binary_authorization}"
```

Comment:
Generally, we do not need to output variables unless there are no other meaningful outputs of a module. This should be dropped.

Reply:
Ok, I'll do it

Reply:
Done